### PR TITLE
Exclude ar_internal_metadata from truncation on Rails 5

### PR DESCRIPTION
**Pull request by hank-spokeo**
Quick fix for the ActiveRecord adapter. This was fixed for the Postgres adapter in Issue  https://github.com/DatabaseCleaner/database_cleaner/issues/445, PR  https://github.com/DatabaseCleaner/database_cleaner/pull/487.

I wrote a spec but had problems building the mysql 2.9.1 gem, so was unable to run the spec (sorry!) but I did load the gem in a project and verified the fix manually.

### CODE REVIEW

File: adapters/database_cleaner-active_record/lib/database_cleaner/active_record/truncation.rb [hunk 1]
```diff
@@ -260,7 +260,10 @@ def tables_to_truncate(connection)
 
     # overwritten
     def migration_storage_names
-      [::DatabaseCleaner::ActiveRecord::Base.migration_table_name]
+      [].tap do |arr|
+        arr << ::DatabaseCleaner::ActiveRecord::Base.migration_table_name
+        arr << ::ActiveRecord::Base.internal_metadata_table_name if ::ActiveRecord::VERSION::MAJOR >= 5
+      end
```
Comment (olleolleolle) [inline adapters/database_cleaner-active_record/lib/database_cleaner/active_record/truncation.rb]: Without losing line count, we could forgo the tap and block:

```ruby
result = [::DatabaseCleaner::ActiveRecord::Base.migration_table_name]
result << ::ActiveRecord::Base.internal_metadata_table_name if ::ActiveRecord::VERSION::MAJOR >= 5
result
```

**Comment by olleolleolle** [comment]
With a rebase, I think this is ready to go.

**Comment by etagwerker** [comment]
@hank-spokeo thank you! I merged it using the alternative suggested by @olleolleolle 👍

**Comment by botandrose** [comment]
@etagwerker @hank-spokeo @olleolleolle Thanks guys! We'll need to get this merged into database_cleaner-active_record, too. The adapter repos are forked off of this one, so I should be able to cherry-pick. I'll give that a go now.

**Pull request merged**